infra/util/pom.xml

```diff
@@ -33,14 +33,66 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.shardingsphere</groupId>
+            <artifactId>shardingsphere-infra-util-groovy</artifactId>
+            <version>${project.version}</version>
+        </dependency>
         
         <dependency>
             <groupId>org.yaml</groupId>
             <artifactId>snakeyaml</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.apache.groovy</groupId>
-            <artifactId>groovy</artifactId>
+            <groupId>org.graalvm.truffle</groupId>
+            <artifactId>truffle-api</artifactId>
         </dependency>
     </dependencies>
+    
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>copy</id>
+                        <goals>
+                            <goal>copy</goal>
+                        </goals>
+                        <phase>package</phase>
```

Comment:
- Logically, we need to advance the Maven life cycle that executes `copy` to `process-test-classes`, so that we can verify the unit tests of Truffle Espresso in the future nativeTest. 

- But I'm not sure how to make `org.apache.shardingsphere:shardingsphere-infra-util-groovy:${project.version}` advance `./mvnw clean install` so that the task of `copy` gets the corresponding JAR.

- Is there any way for Maven to solve this kind of problem without stepping in the CI file?

Reply:
- If the answer to this question is no, considering that the classes that really need to be executed on the Espresso VM are no different on 5.3.1 and 5.3.2, I can use `shardingsphere infra-util:5.3.1` to provide the classpath required for espresso.